server/src/test/java/org/elasticsearch/action/search/SearchAsyncActionTests.java

```diff
@@ -246,10 +253,16 @@ protected SearchPhase getNextPhase(SearchPhaseResults<TestSearchPhaseResult> res
                     return new SearchPhase("test") {
                         @Override
                         public void run() {
-                            latch.countDown();
```

Comment:
Similarly, if we remove this then how do we know the search phase actually runs?

Reply:
yes we are just moving to counting the search the shards rather than the one search, which was counted too early.